resources/qml/TimelineView.qml

```diff
@@ -93,6 +94,23 @@ Page {
 		anchors.fill: parent
 		color: colors.window
 
+		Component {
+			id: deviceVerificationDialog
+			DeviceVerification {}
+		}
+		Connections {
+			target: timelineManager
+			onNewDeviceVerificationRequest: {
+				flow.userId = userId;
+				flow.sender = false;
+				flow.deviceId = deviceId;
+				flow.tranId = transactionId;
+				var dialog = deviceVerificationDialog.createObject(timelineRoot, 
+                    {flow: flow,sender: false});
```

Comment:
any reason you're not using the flow object's sender and are instead re-specifying it?

Reply:
Yeah that's unnecessary